src/api/index.js

```diff
@@ -52,6 +52,7 @@ import getUsers from './users/getUsers';
 import createUser from './users/createUser';
 import getUserProfile from './users/getUserProfile';
 import updateUserStatus from './users/updateUserStatus';
+import getTemporaryFileUrl from './messages/getTemporaryFileUrl';
```

Comment:
nit in commit message: `GET /user_uploads/{realm_id_str}/{filename}` (as opposed to POST, etc.)

Reply:
Done.